packages/components/app/styles/components/pagination.scss

```diff
@@ -10,19 +10,43 @@
 
 .hds-pagination {
   display: grid;
-  grid-template-areas: "left center right";
+  grid-template-areas: "info nav selector";
```

Comment:
I renamed these for clarity and find-ability.

Reply:
@KristinLBradley @cveigt what do you think of this approach/layout? It would optimize even more the vertical space (precious in small screens context) and it's also very simple to achieve:

<img width="314" alt="screenshot_2282" src="https://user-images.githubusercontent.com/686239/213406245-d31c4322-b2c2-44a8-9cbe-005cb44d2ec8.png">